DRAFT: Multi-Party Payout.md

```diff
@@ -0,0 +1,204 @@
+unnecessary to manually add it.
+2. `<payoutStr>`: (DIP3) The Dash address corresponding to the scriptPayout field of the ProRegTx. In case scriptPayout is not a
+P2PK/P2PKH/P2SH script, `<payoutStr>` must be set to the hex representation of the full script.
+3. `<operatorReward>`: (DIP3) The operatorReward field of the ProRegTx.
+4. `<ownerKeyAddress>`: (DIP3) The Dash address corresponding to the keyIdOwner field of the ProRegTx.
+5. `<votingKeyAddress>`: (DIP3) The Dash address corresponding to the keyIdVoting field of the ProRegTx.
+6. `<sharedReward>`: (NEW) the sharedReward field of the ProRegTx.
+7. `<payloadHash>`: (DIP3) The SHA256 hash of the ProRegTx payload with the payloadSig being empty.
+
+The ProRegTx discretely specifies 2 rewards, the operatorReward and the sharedReward. The combined calculation of these two rewards then defines the owner's reward:
+
+* The percentage of the masternode reward paid to the operator is calculated by dividing the operatorReward field by 100.
+* The total percentage of the masternode reward to be paid to shareHolders is calculated by dividing the sharedReward field by 100.
+* The remainder of the masternode reward is paid fully to the owner address.
+* The percentage of the sharedReward paid to each shareHolder is the defined in the shareHolders field.
+
+The ProRegTx also specifies a new field, shareHolders, that contains the addresses and percentage of the sharedReward each address is paid. This field is an array of key:value pairs. The key is Dash addresses that will receive rewards. The value is an integer between 1 and 100 (inclusive). This value is the percentage of the sharedReward the address will receive. To mitigate potential block size issues: this field is limited to a length of 100.
+
+The transaction consists of the following data in the payload area:
+
+| Field | Type | Size | Description | (DIP) |
+| --- | --- | --- | --- | --- |
+| version | uint_16 | 2 | Provider transaction version number.  Currently set to 1. | DIP0003 |
+| type | uint_16 | 2 | Masternode type. Default set to 0. | DIP0003 |
+| mode | uint_16 | 2 | Masternode mode. Default set to 0. | DIP0003 |
+| collateralOutpoint | COutpoint | 36 | The collateral outpoint. | DIP0003 |
+| ipAddress | byte[] | 16 | IPv6 address in network byte order. Only IPv4 mapped addresses are allowed (to be extended in the future) | DIP0003 |
+| port | uint_16 | 2 | Port (network byte order) | DIP0003 |
+| KeyIdOwner | CKeyID | 20 | The public key hash used for owner related signing (ProTx updates, governance voting) | DIP0003 |
+| PubKeyOperator | BLSPubKey | 48 | The public key used for operational related signing (network messages, ProTx updates) | DIP0003 |
+| KeyIdVoting | CKeyID | 20 | The public key hash used for voting. | DIP0003 |
+| operatorReward | uint_16 | 2 | A value from 0 to 10000. | DIP0003 |
+| sharedReward | uint_16 | 2 | A value from 0 to 10000. | current |

```

Comment:
We should just say that, if we are using a specific version, then assume sharedReward is max, otherwise assume sharedReward is 0